.drone.yml

```diff
@@ -8,6 +8,7 @@ trigger:
     include:
       - 'refs/tags/v*'
       - 'refs/heads/main'
+      - 'refs/heads/DOP-4489'
```

Comment:
Formal reminder to remove before merging, just in case

Reply:
done